### PR TITLE
Replace htmlPreview by webview api

### PR DESCRIPTION
**Pull request by qinezh**

### CODE REVIEW

**Comment by jamarw** [comment]
Hi @qinezh , I ran into this error while running `npm run vscode:prepublish`

**src/extension.ts(16,5): error TS2305: Module ''vscode'' has no exported member 'WebviewPanel'.
src/extension.ts(120,26): error TS2339: Property 'createWebviewPanel' does not exist on type 'typeof window'.
src/provider.ts(13,5): error TS2305: Module ''vscode'' has no exported member 'WebviewPanel'.**

If I bump the vscode egine version up to ^1.23.0 the prepublish step runs fine.  Did you run into a similar error?

Thanks

File: docs-preview/src/extension.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ import {
     ViewColumn,
     window,
     workspace,
+    WebviewPanel,
```
Comment (jamarw) [inline docs-preview/src/extension.ts]: Hi @qinezh , I ran into this error while running `npm run vscode:prepublish`

**src/extension.ts(16,5): error TS2305: Module ''vscode'' has no exported member 'WebviewPanel'.
src/extension.ts(120,26): error TS2339: Property 'createWebviewPanel' does not exist on type 'typeof window'.
src/provider.ts(13,5): error TS2305: Module ''vscode'' has no exported member 'WebviewPanel'.**

If I bump the vscode egine version up to ^1.23.0 the prepublish step runs fine.  Did you run into a similar error?

Thanks

Comment (qinezh) [reply docs-preview/src/extension.ts]: You're right! It's required to up vscode engine version to 1.23.0

**Comment by qinezh** [comment]
@jamarw Thanks for the comments. It's much complicated than I thought. As I'm busy on other work items and almost have no more bandwidth to continue work on it recently, is it possible for you to take over it? Thanks~

**Comment by jamarw** [comment]
> @jamarw Thanks for the comments. It's much complicated than I thought. As I'm busy on other work items and almost have no more bandwidth to continue work on it recently, is it possible for you or your team member to take over it? Thanks~

Hi @qinezh , sure we can take it over.  Thanks for the help.

**Comment by bharney-msft** [comment]
Issue #138

**Pull request closed without merging**